RubrikServiceAccount.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+########################################################################################################
+# Title:    Bash-ServiceAccount.sh
+# Summary:  Authenticate BASH using Rubrik service account user.
+# Author:   Parag Bhardwaj
+#
+# REQUIREMENTS:
+# jq is required to parse curl response. You can usually get the jq utility in the add on
+# repository of most linux distros.
+#
+# USAGE: ./Bash-ServiceAccount.sh -c 192.X.X.X -serviceAccountId myServiceAccountID -secret SecretKeyOfTheServiceAccountUser
+########################################################################################################
+
+#Modify the following parameters
+c="X.X.X.X"
+serviceAccountId="User:::5741XXXXXXad6"
+secret="0reaJXXXXXXXX+yWApJF"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- fa00b2ad-9741-498a-845e-6290eb16a257 -->

Reply:
There is no password required for service account config.